src/components/CardPickerCard.js

```diff
@@ -18,6 +18,10 @@ type CardPickerCardProps = {
    * The icon is limited in size to 60x60 pixels.
    */
   defaultIconUri: string | null;
+   /**
+   * The description for scanner type in the card.
+   */
+  desc: string | null;
```

Comment:
We also need to add the description property to the CardPickerCardDefaultProps type (around line 45), and then add it to the defaultProps variable (around line 52) in the class (setting it to null) so the value will be clear if the parent component doesn't set it.